Repository/Configurations/UserConfiguration.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Domain.Models;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Persistence.Configurations
+{
+    class UserConfiguration : IEntityTypeConfiguration<User>
+    {
+        public void Configure(EntityTypeBuilder<User> builder)
+        {
+            builder
```

Comment:
same here, let's agreed to build relations from 1 to many side, not from many to 1